packages/lesswrong/components/users/UsersNameDisplay.tsx

```diff
@@ -58,16 +75,35 @@ const UsersNameDisplay = ({user, color=false, nofollow=false, simple=false, clas
     </span>
   }
 
-  return <span {...eventHandlers} className={className}>
-    <AnalyticsContext pageElementContext="userNameDisplay" userIdDisplayed={user._id}>
-    <LWTooltip title={<UserTooltip user={user}/>} placement={tooltipPlacement} inlineBlock={false}>
-      <Link to={userGetProfileUrl(user)} className={colorClass}
-        {...(nofollow ? {rel:"nofollow"} : {})}
+  const showNewUserIcon = allowNewUserIcon && isNewUser(user);
+  return <span className={className}>
+    <span {...eventHandlers}>
```

Comment:
I have moved this `className` from the inner span around the actual username to this outer span containing the username and icons. I have checked everywhere this is passed in and I'm relatively sure this won't affect the styling elsewhere